email_verification/templates/email_verification/verify.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block title %}Verify email address{% endblock %}
+
+{% block content %}
+<div class="container content">
+    {% if verification.verified %}
+    Thank you for verifying your email address ({{ verification.email }}).
+    {% elif verification.verification_code_expired %}
+    {# TODO: Resend verification email #}
```

Comment:
@omarkhan Leftover TODO, just pointing out to make sure we don't forget about it.